manual/src/connecting.md

```diff
@@ -21,7 +21,7 @@ options.app_name = Some("My App".to_string());
 ```
 
 ## Creating a `Client`
-The `Client` struct is the main entry point for the driver.  You can create one from a `ClientOptions` struct:
+The [`Client`](https://docs.rs/mongodb/latest/mongodb/struct.Client.html) struct is the main entry point for the driver.  You can create one from a `ClientOptions` struct:
```

Comment:
it's sort of performance related so we would probably talk about it more extensively there, but I think it could still be worth briefly mentioning here because it's so important that for most uses cases users should create and persist a single client for their entire app. 
this has been a point of confusion in the past (not just in Rust but for MongoDB drivers in general) by people coming from different DB drivers where the top level object is a single connection

Reply:
Done.